CONTRIBUTING.md

```diff
@@ -1,8 +1,23 @@
 # Contributing
 
-Before contributing to this repository, please try to include the reachability metadata directly into the library. If that does not work, open a ticket on the target library issue tracker so the community can upvote and discuss metadata addition. Only after these steps, follow the checklist for adding the metadata to this repository.
+Before contributing to this repository, please try to include the reachability metadata directly into the library. 
+If that does not work, open a ticket on the target library issue tracker so the community can upvote and discuss metadata addition.
```

Comment:
```suggestion
If that is not an option (for example, you are not a maintainer), we encourage you to open a ticket on the issue tracker of the corresponding library or framework, so that the community can up-vote and discuss the inclusion of reachability metadata.
```